models/elasticsearch.go

```diff
@@ -94,6 +95,11 @@ type ESHighlight struct {
 	DescriptionDatasetID []*string `json:"description.datasetId"`
 }
 
+type Topic struct {
```

Comment:
Both `Topic` and `ContentType` struct are the same. We can make a common struct called `FilterCount` which can then be used for `Topics` and `ContentTypes`

Reply:
Done